builds/misc/images-release.yaml

```diff
@@ -289,7 +289,7 @@ stages:
         - bash: |
             mkdir $(Build.ArtifactStagingDirectory)/publish-linux && \
             mv $(Build.BinariesDirectory)/publish/{CACertificates,e2e_deployment_files,e2e_test_files,\
-            scripts,IotEdgeQuickstart*,LeafDevice*} $(Build.ArtifactStagingDirectory)/publish-linux
+            scripts,IotEdgeQuickstart*,LeafDevice*,*.nupkg} $(Build.ArtifactStagingDirectory)/publish-linux
```

Comment:
How does this .nupkg gets consumed? Does this get pushed to a nuget feed?

Reply:
Right now, it will be published. I'll create another PR to follow up with the consumption of this artifact.